sdk/ml/azure-ai-ml/azure/ai/ml/_schema/compute/usage.py

```diff
@@ -7,7 +7,7 @@
 from marshmallow import fields
 from marshmallow.decorators import post_load
 
-from azure.ai.ml._restclient.v2022_10_01_preview.models import UsageUnit
+from azure.ai.ml._restclient.v2022_12_01_preview.models import UsageUnit
```

Comment:
I think we don't need to change compute schema?

Only change pipeline related will be enough.

Reply:
I revert this change